deploy/roles/eod7/tasks/main.yaml

```diff
@@ -0,0 +1,480 @@
+#   - abyssinica-fonts
+    - tree
+    - expect
+  tags:
+    - eod
+
+#- name: set up RHEL 7 EPEL Repository
+#  become: yes
+#  become_method:  sudo
+#  command:  subscription-manager repo --enable r-epel-rhel-7-x86-64
+
+- include: install_eod.yaml
+  when: eod_install
+  tags:
+    - eod
+
+- include: upgrade_eod.yaml
+  when: eod_upgrade
+  tags:
+    - eod
+
+- name: Configure node.cfg
+  template:
+    src: node.cfg.j2
+    dest: "{{ eod_install_directory }}/conf/admin/node.cfg"
+  when: eod_datastore_directory is defined
+  tags:
+    - eod
+
+- name: Configure cluster.cfg
+  template:
+    src: cluster.cfg.j2
+    dest: "{{ eod_install_directory }}/conf/admin/cluster.cfg"
+  when: custom_repo
+  notify:
+    - restart eod
+  tags:
+    - eod
+
+- name: Create admin user directories
+  file:
+    state: directory
+    path: "{{ eod_install_directory }}/conf/users/{{ item }}"
+    owner: root
+    group: root
+    mode: 0750
+  with_items: "{{ eod_admins }}"
+  tags:
+    - eod
+
+- name: Ensure admins can login via Connection Server
+  template:
+    src: admin_settings.cfg.j2
+    dest: "{{ eod_install_directory }}/conf/users/{{ item }}/settings.cfg"
+  with_items: "{{ eod_admins }}"
+  tags:
+    - eod
+
+- name: Configure xdmcp
+  template:
+    src: XDMCP_Query.cfg
+    dest: "{{ eod_install_directory }}/conf/admin/Xconfig/XDMCP_Query.cfg"
+  notify:
+    - restart eod
+    - restart gdm
+#   - restart prefdm
+  when: custom_repo
+  tags:
+    - eod
+
+- name: Configure EOD license file
+  template:
+    src: akeys
+    dest: "{{ eod_install_directory }}/conf/admin/akeys"
+  when: custom_repo
+  notify:
+    - restart eod
+  tags:
+    - eod
+
+- name: Configure PAM authentication
+  template:
+    src: exceed-connection-server
+    dest: /etc/pam.d/exceed-connection-server
+    mode: 0644
+  notify:
+    - restart eod
+  tags:
+    - eod
+
+# TODO: Add template for node.cfg
+
+### Install and configure Desktop Environment
+
+#- name: install X, Desktop, fonts
+#  command: yum -y groupinstall gnome-desktop x11 fonts
+
+#- name: install gdm (gnome display manager)
+#  command: yum -y install gdm
+  
+#- name: install X Window System
+#  yum:
+#    name: "@X Window System"
+#    state: present
+#    state: latest
+# skip_broken: yes
+#  tags:
+#    - eod
+
+- name: install "X Window System 
+  shell: yum -y groupinstall "X Window System"
+
+#- name: install Desktop
+# shell: yum -y groupinstall X11  
+ 
+#  shell: yum -y groupinstall X11 --setopt=group_package_types=mandatory,default,optional
+#  shell: yum -y groupinstall gnome-desktop x11 fonts --skip-broken
+#  shell: yum -y groupinstall "gnome-desktop"
+#  yum:
+#    name: "@gnome-desktop"
+#    state: present
+# skip_broken: yes
+  tags:
+    - eod
+
+#- name: install gnome-backgrounds
+#  yum:
+#    name: gnome-backgrounds
+#    state: present
+#  tags:
+#    - eod
+  
+- name: install gdm (gnome display manager)
+  command: yum -y install gdm
+  tags:
+    - eod
```

Comment:
@lel99999 Doesn't xfce use a different display manager?

Reply:
had to look up for clarity and preciseness: it appears xfce is a DE (Desktop Environment) based on gtk and has a native WM (Windows Manager - handles/controls placement and decoration of windows), while gdm/kdm/xdm are DMs (Display Managers - synonymous with login managers, first X program run by X system for login), so XFCE can use gdm or lightdm